--- dlopen-0.1.8/Cargo.toml	1970-01-01T00:00:00+00:00
+++ dlopen-0.1.8/Cargo.toml	2021-04-29T01:28:46.664152+00:00
@@ -61,9 +61,6 @@
 version = "0.2"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.29"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.7"
-features = ["winnt", "minwindef", "winerror", "libloaderapi", "errhandlingapi", "dbghelp", "processthreadsapi", "basetsd"]
 [badges.appveyor]
 branch = "master"
 repository = "szymonwieloch/rust-dlopen"
