--- fslock-0.1.6/Cargo.toml	2020-04-12T18:49:49+00:00
+++ fslock-0.1.6/Cargo.toml	2021-04-17T13:57:41.665165+00:00
@@ -28,9 +28,6 @@
 [target."cfg(unix)".dependencies.libc]
 version = "^0.2.66"
 default-features = false
-[target."cfg(windows)".dependencies.winapi]
-version = "^0.3.8"
-features = ["minwindef", "minwinbase", "winbase", "errhandlingapi", "winerror", "winnt", "synchapi", "handleapi", "fileapi"]
 [badges.maintenance]
 status = "passively-maintained"
 
