--- deno_lint-0.4.0/Cargo.toml	1970-01-01T00:00:00+00:00
+++ deno_lint-0.4.0/Cargo.toml	2021-04-29T17:20:05.410699+00:00
@@ -44,7 +44,7 @@
 version = "1.5.2"
 
 [dependencies.regex]
-version = "=1.4.3"
+version = "1.4.3"
 
 [dependencies.serde]
 version = "1.0.125"
@@ -70,7 +70,7 @@
 version = "2.33.3"
 
 [dev-dependencies.deno_core]
-version = "0.84.0"
+version = "0.86.0"
 
 [dev-dependencies.env_logger]
 version = "0.8.3"
