--- deno-1.13.2/Cargo.toml	1970-01-01T00:00:01+00:00
+++ deno-1.13.2/Cargo.toml	2021-09-04T16:56:20.167305+00:00
@@ -259,14 +259,3 @@
 
 [target."cfg(unix)".dev-dependencies.nix]
 version = "0.20.0"
-[target."cfg(windows)".dependencies.fwdansi]
-version = "1.1.0"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-features = ["knownfolders", "mswsock", "objbase", "shlobj", "tlhelp32", "winbase", "winerror", "winsock2"]
-[target."cfg(windows)".build-dependencies.winapi]
-version = "0.3.9"
-
-[target."cfg(windows)".build-dependencies.winres]
-version = "0.1.11"
