--- password-hash-0.3.2/Cargo.toml	1970-01-01T00:00:01+00:00
+++ password-hash-0.3.2/Cargo.toml	2021-12-13T17:22:30.987558+00:00
@@ -25,7 +25,7 @@
 all-features = true
 rustc-args = ["--cfg", "docsrs"]
 [dependencies.base64ct]
-version = ">=1, <1.1.0"
+version = "1"
 
 [dependencies.rand_core]
 version = "0.6"
@@ -33,7 +33,7 @@
 default-features = false
 
 [dependencies.subtle]
-version = ">=2, <2.5"
+version = "2"
 default-features = false
 
 [features]
