--- crypto-bigint-0.2.11/Cargo.toml	1970-01-01T00:00:01+00:00
+++ crypto-bigint-0.2.11/Cargo.toml	2021-12-10T16:22:13.642298+00:00
@@ -32,17 +32,12 @@
 version = "0.6"
 optional = true
 
-[dependencies.rlp]
-version = "0.5"
-optional = true
-default-features = false
-
 [dependencies.subtle]
 version = "2.4"
 default-features = false
 
 [dependencies.zeroize]
-version = ">=1, <1.5"
+version = "1"
 optional = true
 default-features = false
 [dev-dependencies.hex-literal]
