--- gfx-backend-gl-0.9.0/Cargo.toml	1970-01-01T00:00:00+00:00
+++ gfx-backend-gl-0.9.0/Cargo.toml	2021-09-06T23:01:26.674686+00:00
@@ -75,12 +75,3 @@
 
 [target."cfg(not(target_arch = \"wasm32\"))".dependencies.libloading]
 version = "0.7"
-[target."cfg(target_arch = \"wasm32\")".dependencies.js-sys]
-version = "0.3.6"
-
-[target."cfg(target_arch = \"wasm32\")".dependencies.wasm-bindgen]
-version = "0.2.60"
-
-[target."cfg(target_arch = \"wasm32\")".dependencies.web-sys]
-version = "0.3.6"
-features = ["console", "Document", "Element", "HtmlCanvasElement", "HtmlElement", "Node", "Performance", "WebGlBuffer", "WebGlRenderingContext", "WebGl2RenderingContext", "WebGlProgram", "WebGlSampler", "WebGlShader", "WebGlTexture", "Window"]
